awsbestpractices/sqs-name-vs-url/SqsNameVsUrl.java

```diff
@@ -0,0 +1,18 @@
+package com.amazon.promotions.stream.dlq;
+
+import com.amazonaws.services.sqs.model.Message;
+
+public class SqsNameVsQueue implements Callable<Message> {
+    public void main() {
+        // Name instead of url
+        sqsMetricsClient.changeMessageVisibility(queueName, "foo", "bar");
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

This code does not provide the full URL for the Amazon SQS queue. The name alone might not be sufficient. Consider providing the full URL for the queue.